src/utils/requests/BillRequests.js

```diff
@@ -3,10 +3,10 @@ const {URL} = require("../../config");
  * @function getBill
  * @description Function that calls the /bills route to return the bills associated to the user. With bearer token.
  */
-export const getBill = async () => {
+export const getBill = async (query_params) => {
```

Comment:
use default params, so you don't need to define param

```suggestion
export const getBill = async (query_params = '') => {
```